modules/tinymce/src/plugins/importcss/main/ts/core/ImportCss.ts

```diff
@@ -255,7 +257,7 @@ const setup = (editor: Editor): void => {
     };
 
     Tools.each(getSelectors(editor, editor.getDoc(), compileFilter(Settings.getFileFilter(editor))), (selector) => {
-      if (selector.indexOf('.mce-') === -1) {
+      if (selector.match(internalEditorStyle) === null) {
```

Comment:
```suggestion
      if (!internalEditorStyle.test(selector)) {
```
`test` is faster in most cases than `match` and we don't care about using the match group, so it avoids the `null` check and the various points around it.

Reply:
right, I had a vague memory there was a better way but forgot with everything else going on. Thanks!